### ISSUE TITLE
Problems building on OpenBSD

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#439]
Originally reported on Google Code with ID 440

```
Attempting to do a default install of mupen64plus-1.99.4 on OpenBSD amd64. Several problems
pop up:

- The m64p_*.sh scripts have #!/bin/bash shebangs. OpenBSD has bash installed at /usr/local/bin/bash
(and it’s not installed by default). But these scripts work just fine with #!/bin/sh.
A patch for this is attached.

- The m64p_*.sh scripts call make directly. The makefiles aren’t compatible with BSD
make, and on OpenBSD GNU make is installed as gmake. It would be convenient to change
the make variant with an environment variable or something.

- OS detection scripts fail because there is no OpenBSD entry. Things seemed to work
okay if I just copied FreeBSD’s, but I’m not sure how good of an idea this is. A patch
of what I did is attached.

- screenshot.cpp doesn’t see png.h, which is located at /usr/local/include/libpng/png.h.
A workaround is setting CXXFLAGS=-I/usr/local/include/libpng before building.

- Running mupen64plus from the test/ directory doesn’t seem to work: I get a black
screen. Then when I hit ^C a double free occurs. The output log is attached.
```

Reported by `anthonyjbentley` on 2011-06-13 05:22:43

<hr>
- _Attachment: [patch-bin-sh.diff](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-440/comment-0/patch-bin-sh.diff)_
- _Attachment: [patch-openbsd-freebsd.diff](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-440/comment-0/patch-openbsd-freebsd.diff)_
- _Attachment: [mupen.log](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-440/comment-0/mupen.log)_

**Comment by richard42** [comment]
```
Sorry, dupe of #414.
```

Reported by `anthonyjbentley` on 2011-06-25 03:57:50

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2011-06-25 06:39:45
- **Status changed**: `Duplicate`
- **Merged into**: #414
